contracts/Comptroller.sol

```diff
@@ -501,6 +504,10 @@ contract Comptroller is ComptrollerV5Storage, ComptrollerInterfaceG2, Comptrolle
         // Shh - currently unused
         liquidator;
```

Comment:
It's used now, no need to silence the warning

Reply:
Fixed in [a35afda](https://github.com/VenusProtocol/venus-protocol/pull/88/commits/a35afda6e607cb7c155f96ef065128349c1ccf1a)